.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     "@typescript-eslint/no-unsafe-member-access": "off",
     "@typescript-eslint/no-explicit-any": "off",
     "@typescript-eslint/no-unsafe-call": "off",
-    "@typescript-eslint/no-unsafe-assignment": "off"
-
+    "@typescript-eslint/no-unsafe-assignment": "off",
+    "@typescript-eslint/no-inferrable-types": "off",
```

Comment:
this might cause issues down the line, but why not

Reply:
This is something Ryan added, so I have no idea what it is..